lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -223,6 +224,59 @@ describe('modules/manager/npm/extract/index', () => {
       expect(res?.npmrc).toBe('registry=https://registry.npmjs.org\n');
     });
 
+    describe('yarnrc support', () => {
+      it('reads .yarnrc.yml', async () => {
+        fs.readLocalFile = jest.fn((fileName) => {
+          if (fileName === '.yarnrc.yml') {
+            return 'npmRegistryServer: https://registry.example.com';
+          }
+          return null;
+        });
+        const res = await npmExtract.extractPackageFile(
+          input02Content,
+          'package.json',
+          {}
+        );
+        expect(
+          res?.deps.flatMap((dep) => dep.registryUrls)
+        ).toBeArrayIncludingOnly(['https://registry.example.com']);
```

Comment:
```suggestion
        ).toEqual(['https://registry.example.com']);
```
isn't this the same?

Reply:
Nope, because the array has multiple entries (one for each flat-mapped dependency)